### PR TITLE
Gametest2

### PR DESCRIPTION
**Pull request by stubailo**

### CODE REVIEW

**Comment by stubailo** [comment]
e4bed1c9e4df86fb19b664ba1f4a8d870379f8da fixed the game.players bug

^^ is there a way to pull that into the branch? or a way to pull all of the changes in develop in?

File: db/schema.rb [hunk 1]
```diff
@@ -65,6 +65,7 @@
     t.string   "unconfirmed_email"
     t.datetime "created_at"
     t.datetime "updated_at"
+    t.integer  "games_count"
```
Comment (vpontis) [inline db/schema.rb:68]: where is this coming from? and why would we want this field? @santhoshnarayan

File: test/models/game_test.rb [hunk 2]
```diff
@@ -23,4 +23,70 @@ class GameTest < ActiveSupport::TestCase
     assert g.winners.count == 2, 'The winners didnt save!'
     assert g.losers.count == 2, 'The losers didnt save!'
   end
+
+  test "players" do
+    g = Game.new
+    g.winners = [players(:a), players(:b)]
+    g.losers = [players(:c), players(:d)]
+    g.save
+    assert g.players.map {|p| p.username} == [players(:a), players(:b), players(:c), players(:d)], g.players.map {|p| p.username}
```
Comment (vpontis) [inline test/models/game_test.rb:32]: why are you not using g.player_usernames?

Comment (stubailo) [inline test/models/game_test.rb:32]: I'm pretty sure that method doesn't exist, although we should probably add it

File: test/models/game_test.rb [hunk 3]
```diff
@@ -23,4 +23,70 @@ class GameTest < ActiveSupport::TestCase
...
+    g = Game.new
+    g.winners = [players(:a), players(:b)]
+    g.losers = [players(:c), players(:d)]
+    g.save
+    g.confirm
+
+    assert_equal g.confirmed?, true, "game was not confirmed, should have been"
+  end
+
+  # test "all users found" do
```
Comment (vpontis) [inline test/models/game_test.rb:75]: try not to push commented code

File: test/models/game_test.rb [hunk 4]
```diff
@@ -23,4 +23,70 @@ class GameTest < ActiveSupport::TestCase
...
+    g = Game.new
+    g.winners = [players(:a), players(:b)]
+    g.losers = [players(:c), players(:d)]
+    g.confirmed = true
+    g.save
+
+    assert_equal g.confirmed?, true, "game was not confirmed, should have been"
+  end
+
+  test "confirm" do
```
Comment (vpontis) [inline test/models/game_test.rb:64]: try and be a little more descriptive with the tests

File: test/models/game_test.rb (hunk 2 above)
Comment (santhoshnarayan) [inline test/models/game_test.rb:32]: yeah didnt exist

File: test/models/game_test.rb (hunk 3 above)
Comment (santhoshnarayan) [inline test/models/game_test.rb:75]: just there as a reminder to add it but kk

File: test/models/game_test.rb (hunk 4 above)
Comment (santhoshnarayan) [inline test/models/game_test.rb:64]: kk

**Pull request closed without merging**